test/unit/vl-select.test.html

```diff
@@ -399,8 +411,91 @@
+        const select = fixture('vl-select-auto-dress-fixture');
+        assert.equal(vl.i18n.i18n['select.no_more_options'], select.DEFAULT_NO_MORE_OPTIONS);
+        assert.equal(vl.i18n.i18n['select.no_results'], select.DEFAULT_SEARCH_NO_RESULT);
+        assert.equal(vl.i18n.i18n['select.search_placeholder_value'], select.DEFAULT_SEARCH_PLACEHOLDER);
+      });
+
+      test('de placeholder tekst kan per select geconfigureerd worden', async () => {
+        const selects = fixture('multiple-selects-fixture');
+        assert.equal(await getPlaceholderText(selects[0]), selects[0].getAttribute('data-vl-search-placeholder'));
+        assert.equal(await getPlaceholderText(selects[1]), selects[1].getAttribute('data-vl-search-placeholder'));
+        assert.notEqual(await getPlaceholderText(selects[0]), await getPlaceholderText(selects[1]));
+      });
+
+      test('de geen resultaten tekst kan per select geconfigureerd worden', async () => {
+        const selects = fixture('multiple-selects-fixture');
+        const geenResultatenText1 = await getNoResultsText(selects[0]);
+        const geenResultatenText2 = await getNoResultsText(selects[1]);
+        assert.equal(geenResultatenText1, selects[0].getAttribute('data-vl-search-no-results-text'));
+        assert.equal(geenResultatenText2, selects[1].getAttribute('data-vl-search-no-results-text'));
+        assert.notEqual(geenResultatenText1, geenResultatenText2);
+      });
+
+      test('de geen keuzes meer tekst kan per select geconfigureerd worden', async () => {
+        const selects = fixture('multiple-selects-fixture');
+        assert.equal(await getNoMoreOptionsText(selects[0]), selects[0].getAttribute('data-vl-no-more-options'));
+        assert.equal(await getNoMoreOptionsText(selects[1]), selects[1].getAttribute('data-vl-no-more-options'));
+        assert.notEqual(await getNoMoreOptionsText(selects[0]), await getNoMoreOptionsText(selects[1]));
+      });
+
+      test('de teksten configuratie van een select heeft geen invloed op een select zonder tekst configuratie die de standaard teksten zal tonen', async () => {
+        const selects = fixture('multiple-selects-default-texts-fixture');
+        assert.equal(await getPlaceholderText(selects[0]), selects[0].getAttribute('data-vl-search-placeholder'));
+        assert.equal(await getPlaceholderText(selects[1]), selects[1].DEFAULT_SEARCH_PLACEHOLDER);
+        assert.notEqual(await getPlaceholderText(selects[0]), await getPlaceholderText(selects[1]));
+
+        const geenResultatenText1 = await getNoResultsText(selects[0]);
+        const geenResultatenText2 = await getNoResultsText(selects[1]);
+        assert.equal(geenResultatenText1, selects[0].getAttribute('data-vl-search-no-results-text'));
+        assert.equal(geenResultatenText2, selects[1].DEFAULT_SEARCH_NO_RESULT);
+        assert.notEqual(geenResultatenText1, geenResultatenText2);
```

Comment:
```suggestion
        assert.equal(await getNoResultsText(selects[0]), selects[0].getAttribute('data-vl-search-no-results-text'));
        assert.equal(await getNoResultsText(selects[1]), selects[1].DEFAULT_SEARCH_NO_RESULT);
        assert.notEqual(await getNoResultsText(selects[0]), await getNoResultsText(selects[1]));
```

Reply:
De oorzaak van deze wijziging is vermoedelijk gebeurd omdat 2 x `getNoResultsText` oproepen niet hetzelfde resultaat geeft. Ik heb het achterliggend probleem opgelost, waardoor de const declaratie niet meer nodig is, maar voornamelijk om problemen in de toekomst te voorkomen wanneer meermaals `getNoResultsText` opgeroepen moet worden.